bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/common/registry/AbstractRegistry.java

```diff
@@ -139,6 +139,7 @@ public void addRegistryChangeListener(RegistryChangeListener<E> listener) {
         listeners.add(listener);
     }
 
+    @SuppressWarnings("null")
```

Comment:
I'd be happy if you'd find a better solution - my issue is that the whole Java8 Stream API is pretty much incompatible with those annotations. Which might be a hint that for all new stream based APIs that we introduce, we might indeed want to consider the use of `Optional`.